### PR TITLE
Use sbt-web 1.5.x despite staying on paradox 0.9.2

### PR DESCRIPTION
**Pull request by mkurz**
Same as in 
- https://github.com/apache/incubator-pekko-http/pull/486
- https://github.com/apache/incubator-pekko/pull/1135
- https://github.com/apache/incubator-pekko-sbt-paradox/pull/114

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -7,8 +7,6 @@
  * This file is part of the Apache Pekko project, which was derived from Akka.
  */
 
-dependencyOverrides += "org.scala-lang.modules" %% "scala-xml" % "2.2.0"
```
Comment (mkurz) [inline project/plugins.sbt:10]: Seems this override is not needed anymore. I think this was from the days

**Comment by mdedetrich** [comment]
Hmm, tests are failing

**Comment by mkurz** [comment]
> Hmm, tests are failing

Seems this is not related to my pull request. [Latest commits](https://github.com/apache/incubator-pekko-connectors/commits/main/) in `main` fail with the same errors:
```
[error] import javax.jms
[error]        ^

not found: value jms
```

**Comment by pjfanning** [comment]
see https://github.com/apache/incubator-pekko-connectors/pull/514

**Comment by pjfanning** [comment]
I rebased this with the jms build fix

**Review by pjfanning — APPROVED** [review]
lgtm

**Pull request merged**